### PR TITLE
fix: databricks/odbc PR (#3409) breaking cli install 

### PR DESCRIPTION
**Pull request by IrakliJani**
fixes: https://github.com/lightdash/lightdash/issues/3398
### Description:
#3409 was breaking CLI install
published https://github.com/databricks/databricks-sql-nodejs/commit/73874d4b14a0ffb5abeb9c46d338696a2b408671 as https://www.npmjs.com/package/@lightdash/databricks-sql
created issue on the original repo: https://github.com/databricks/databricks-sql-nodejs/issues/78

### CODE REVIEW

File: packages/warehouses/package.json [hunk 1]
```diff
@@ -10,26 +10,26 @@
     "dependencies": {
         "@google-cloud/bigquery": "^5.9.1",
         "@lightdash/common": "^0.291.1",
-        "snowflake-sdk": "^1.6.4",
+        "@lightdash/databricks-sql": "^0.1.8-beta.3",
```
Comment (rephus) [inline packages/warehouses/package.json:13]: should we keep beta version ?

**Review by rephus — COMMENTED** [review]
Looks good,l could you try pushing a `beta` or `RC` version for lightdash/cli so we can test this ?

File: packages/warehouses/package.json (hunk 1 above)
Comment (IrakliJani) [reply packages/warehouses/package.json:13]: @rephus as the original package is with beta label - I think yes. we can keep it that way until they release a new version and then we can switch back to their original package.

**Comment by IrakliJani** [comment]
@rephus
> Looks good,l could you try pushing a `beta` or `RC` version for lightdash/cli so we can test this ?

yep, good idea

**Comment by IrakliJani** [comment]
@rephus yep, it works now

**Comment by IrakliJani** [comment]
@rephus yep, it works now

**Pull request merged**